test/integration/max_request_limit.js

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import chaiAsPromised from 'chai-as-promised';
+import ServerApparatus from '../helpers/server_apparatus';
+import {properAddress} from '../helpers/web3chai';
+
+chai.use(properAddress);
+chai.use(chaiHttp);
+chai.use(chaiAsPromised);
+
+const {expect} = chai;
+
+describe('Request size limit test', async () => {
+  let apparatus;
+  const requestSizeLimit = 123;
+
+  const generateStringOfSize = (size) => Array(size)
+    .fill('0')
+    .join('');
+
+  before(async () => {
+    apparatus = new ServerApparatus({requestSizeLimit});
+    await apparatus.start(null);
+  });
+
+  beforeEach(async () => {
+    await apparatus.cleanDB();
+  });
+
+  it('should not accept requests bigger than requestSizeLimit', async () => {
+    const tooLongData = generateStringOfSize(requestSizeLimit + 1);
+    const request = apparatus.request()
+      .post('/accounts')
+      .type('json')
+      .send(tooLongData);
+    await expect(request).to.be.eventually.rejected.and.have.property('status', 413);
+  });
+
+  it('should accept requests under requestSizeLimit', async () => {
+    const data = generateStringOfSize(requestSizeLimit);
+    const request = apparatus.request()
+      .post('/accounts')
+      .type('json')
+      .send(data);
+    await expect(request).to.be.eventually.rejected.and.have.property('status', 500);
```

Comment:
A) What a particularly interesting way to check an expectation. I would take something that actually accepts some data, send it, and await a 200
B) If you find a endpoint that returns `500` for whatever reason. Please feel obligated to fix it right away 😛